api/source/controllers/Asset.js

```diff
@@ -597,3 +597,116 @@ module.exports.updateAsset = async function updateAsset (req, res, next) {
   }
 }
 
+
+async function getAssetIdAndCheckPermission(request) {
+  const elevate = request.query.elevate
+  let assetId = request.params.assetId
+
+  // fetch the Asset for access control checks and the response
+  let assetToAffect = await Asset.getAsset(assetId, [], elevate, request.userObject)
```

Comment:
Please replace the `elevate` param with `undefined` or `false`. This method will be reworked soon to remove that argument entirely, because Admin users cannot elevate their requests for Asset objects

Reply:
set to false